packages/legend-query/src/stores/QueryBuilderPropertyEditorState.ts

```diff
@@ -321,6 +444,20 @@ export class QueryBuilderPropertyExpressionState {
       ) {
         requiresExistsHandling = true;
       }
+      if (
```

Comment:
Let's add a comment here to explain that we're checking if the property is milestoned